YIF_Backend/Controllers/LectorController.cs

```diff
@@ -9,6 +14,26 @@ namespace YIF_Backend.Controllers
     [Authorize(Roles = "Lector")]
     public class LectorController : ControllerBase
     {
+        private readonly ILectorService _lectorService;
+        public LectorController(ILectorService lectorService)
+        {
+            _lectorService = lectorService;
+        }
 
+        /// <summary>
+        /// Get all departments.
+        /// </summary>
+        /// <returns>List of departments</returns>
+        /// <response code="200">Returns a list of departments</response>
+        /// <response code="404">If there are no departments</response>
```

Comment:
add response code "401" If user is unauthorized, token is bad/expired 
and response code "403" if user is not lector

Reply:
Done